fs-repo-14-to-15/main.go

```diff
@@ -0,0 +1,11 @@
+package main
+
+import (
+	mg14 "github.com/ipfs/fs-repo-migrations/fs-repo-14-to-15/migration"
+	migrate "github.com/ipfs/fs-repo-migrations/tools/go-migrate"
+)
+
+func main() {
+	m := mg14.Migration{}
```

Comment:
14 ? I don't remember if we use the previous or future version id.

Reply:
@Jorropo on `fs-repo-13-to-14` it says `mg13`.